.github/workflows/trunk-hourly-e2e-tests.yml

```diff
@@ -126,9 +125,12 @@ jobs:
 
       - run: ls -l ./cypress/reports/mochareports/
       - name: Upload results
-        run: ./.trunk-ci-analytics/bin --token=${{ secrets.TRUNK_DEBUGGER_TOKEN }}  --input="cypress/reports/**/*junit*.xml"
+        uses: trunk-io/trunk-analytics-uploader@v0.1.0
+        with:
+          junit_paths: cypress/reports/**/*junit*.xml
+          org_url_slug: trunk-staging-org
+          token: ${{ secrets.TRUNK_DEBUGGER_TOKEN }}
         env:
-          REPO_URL: ${{ github.repositoryUrl }}
```

Comment:
Will the CLI auto-detect this or do we need to keep the REPO_URL env var here

Reply:
Yeah, env is not needed. Cli should auto-discover this URL.